AirSim/setup.sh

```diff
@@ -30,8 +30,10 @@ if [ "$VERSION" -lt "20" ]; then
     sudo apt-get install -y clang-8 clang++-8 libc++-8-dev libc++abi-8-dev
 elif [ "$VERSION" -lt "22" ]; then
     sudo apt-get install -y clang-8 libc++-8-dev libc++abi-8-dev
-else
+elif [ "$VERSION" -lt "24" ]; then
```

Comment:
If you want to easily install clang and all other tools without checking the Ubuntu Version, you may want to have a look at the llvm script (https://apt.llvm.org/). This will download and install everything on all supported ubuntu versions. This could be easier instead of this if-else construct:

Reply:
That's indeed an option, but that script needs to be downloaded and changes from time to time, for instance clang8 used to be able to be installed with it, but this is no longer the case (because of deprecation etc). The else if case now guarantees that a clang version is installed that is available as apt packages. But I will keep an else clause in case none of the other ones apply.

Reply:
Oh yes, you're right with the clang version 8. Sounds like a good solution for me with the else clause👍